guardian/grid
# IMAGEDAM-1666: Photosales icon is visible when syndication usage is present

## What does this change?

**Note: this PR should be reviewed after #4267 as it builds on the changes included in that PR**

The relevant files for this PR are:
- kahuna/public/js/components/gr-icon/gr-icon.css
- kahuna/public/js/components/gr-icon/gr-icon.js
- kahuna/public/js/components/gr-icon/icons/sent-to-photosales.svg
- kahuna/public/js/preview/image.html
- kahuna/public/js/preview/image.js
- kahuna/public/js/services/image/usages.js

As an archivist user of BBC Images, after I have sent an image to BBC Photo Sales I want to see a visual indication that this has taken place. Therefore, after an image has been selected and sent to Photo Sales a 'sales' icon should appear in the thumbnail of that image.

The Photosales functionality is BBC specific, hence it is disabled by default via the showSendToPhotoSales feature flag.

## How should a reviewer test this change?

As a user with elevated permissions and with the showSendToPhotoSales flag set to true:

A reviewer can test this change by selecting an image and indicating that they want to send it to BBC Photo Sales. After this action has completed, a 'SALES' icon should appear in the thumnbail of that image, as demonstrated in the following screenshot:

![Screenshot 2024-05-13 at 10 20 21](https://github.com/guardian/grid/assets/129299738/37faf18a-f625-4ab2-b586-26484be6361b)



## How can success be measured?

This testing can be considered successful if after following the above steps, the 'SALES' icon appears on the selected image, as demonstrated in the previous screenshot. Additionally, images without a syndication usage should not have this icon present.  

## Who should look at this?


## Tested? Documented?
